packages/loader/container-loader/.eslintrc.cjs

```diff
@@ -17,7 +14,7 @@ module.exports = {
 			files: ["*.spec.ts", "src/test/**"],
 			rules: {
 				// Test files are run in node only so additional node libraries can be used.
-				"import/no-nodejs-modules": ["error", { allow: ["assert"] }],
+				"import/no-nodejs-modules": ["error", { allow: ["node:assert"] }],
```

Comment:
For reviewers: one of the rules in the recommended config requires prepending node dependency imports with `node:`. This updates the existing special rule allowance to accommodate the new import patterns.

Reply:
Can you elaborate on import patterns? Are these patterns relevant to linter rules on order of imported package, spacing between the import statements, and etc.?

Reply:
It's specifically a rule that requires we preface imports from built-in node libraries with `node:`. E.g. `node:assert` instead of `assert`.